<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>registerWindow</class>
 <widget class="QWidget" name="registerWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>677</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>480</width>
    <height>677</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>480</width>
    <height>677</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>新用户注册</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background:rgb(243, 248, 254)</string>
  </property>
  <widget class="QLabel" name="Title">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>90</y>
     <width>271</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
font: 700 26pt &quot;Microsoft YaHei UI&quot;;
color:rgb(0, 157, 255);</string>
   </property>
   <property name="text">
    <string>新用户注册</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="LoginButton">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>490</y>
     <width>211</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	background:rgb(7, 148, 255);
	border-radius:5px;
	color: white;
	font: 18pt &quot;微软雅黑&quot;;
}</string>
   </property>
   <property name="text">
    <string>注      册</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="PassWordLine">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>230</y>
     <width>211</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit{
	background:rgb(255, 255, 255);
	color: black;
	border-radius:5px;
	
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="placeholderText">
    <string>设置密码</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="UserNameLine">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>170</y>
     <width>211</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit{
	background:rgb(255, 255, 255);
	color: black;
	border-radius:5px;
	
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="placeholderText">
    <string>请设置用户名</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="PassWordLine_2">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>290</y>
     <width>211</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit{
	background:rgb(255, 255, 255);
	color: black;
	border-radius:5px;
	
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="placeholderText">
    <string>再次确认密码</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="phoneLine">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>350</y>
     <width>211</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit{
	background:rgb(255, 255, 255);
	color: black;
	border-radius:5px;
	
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="placeholderText">
    <string>输入邮箱/手机号</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="verLine">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>410</y>
     <width>211</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit{
	background:rgb(255, 255, 255);
	color: black;
	border-radius:5px;
	
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="placeholderText">
    <string>验证码</string>
   </property>
  </widget>
  <widget class="QLabel" name="verLabel">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>411</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Matura MT Script Capitals</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="BackButton">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>550</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 170, 255);</string>
   </property>
   <property name="text">
    <string>返回登录 </string>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="senote">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>330</y>
     <width>191</width>
     <height>16</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="vernote">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>450</y>
     <width>211</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="namenote">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>210</y>
     <width>191</width>
     <height>16</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="passnote">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>270</y>
     <width>191</width>
     <height>16</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="idnote">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>390</y>
     <width>191</width>
     <height>16</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>BackButton</tabstop>
  <tabstop>UserNameLine</tabstop>
  <tabstop>PassWordLine</tabstop>
  <tabstop>PassWordLine_2</tabstop>
  <tabstop>phoneLine</tabstop>
  <tabstop>verLine</tabstop>
  <tabstop>LoginButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
